site/src/components/indicator/forceIndex.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: component
+title: Force Index
+component: indicator/algorithm/forceIndex.js
+tags:
+namespace: indicator
+
+example-code: |
+
+  // Create and apply the Force Index algorithm
+  var forceAlgorithm = fc.indicator.algorithm.forceIndex();
+  forceAlgorithm(data);
+  
+  // Calculate the maximum absolute value of the index
+  var aboluteMax = Math.max(Math.abs(d3.min(data, function(d) { return d.force; })),
+    d3.max(data, function(d) { return d.force; }));
+  
+  //Scailing the display using the maximum absolute value of the Index
+  var yScale = d3.scale.linear()
+          .domain([-aboluteMax, aboluteMax])
+          .range([height - 10, 10]);
+
+  // Create the renderer
+  var force = fc.indicator.renderer.forceIndex()
+      .xScale(xScale)
+      .yScale(yScale);
+
+  // Add it to the container
+  container.append('g')
+      .datum(data)
+      .call(force);
+---
+
+A [Force Index](http://en.wikipedia.org/wiki/Force_index) is an indicator that illustrates how strong the actual buying
+or selling pressure is. It is calculated by subtracting the today's closing price from the yesterday's one and
```

Comment:
It is calculated by subtracting ~~the~~ today's closing price from ~~the~~ yesterday's one and multiplying the result by today's volume.